src/test/java/org/jabref/logic/bibtexkeypattern/BibtexKeyGeneratorTest.java

```diff
@@ -73,7 +73,7 @@ void testCrossrefAndInAuthorNames() throws Exception {
 
         assertEquals("Holland",
                 BibtexKeyGenerator.cleanKey(BibtexKeyGenerator.generateKey(entry1, "auth",
-                        database), true));
+                        database), "-`ʹ:!;?^+"));
```

Comment:
Please extract `"-`ʹ:!;?^+"` to a public constant `DEFAULT_UNWANTED_CHARACTERS` in `BibtexKeyGenerator` and use it everywhere.

Reply:
I wasn't sure it was correct, thanks!

Reply:
It's usually a good idea to don't have duplication of code or magic strings. Makes it harder to maintain in the future.